es-vue-base/src/lib-utils/nav-bar-global-content.js

```diff
@@ -395,7 +395,7 @@ export default (
                 },
                 {
                     name: 'Get quotes for solar + batteries',
-                    link: `${ES_DOMAIN}/market/start/`,
+                    link: `${ES_DOMAIN}/shop/home-solar/`,
```

Comment:
We still want this "Get quotes for solar + batteries" link to point to `/market/start/`, as that's meant to be a direct way for customers to start the signup process.

Reply:
Thanks! Done.